### PR TITLE
Initial commit of GLES2 engine

### PR DESCRIPTION
**Pull request by szledan**
Signed-off-by: Kristof Kosztyo <kkristof@inf.u-szeged.hu>
Signed-off-by: Szilard Ledan <szledan@gmail.com>

### CODE REVIEW

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,5 @@ function(build_example directory)
 endfunction(build_example)
 
 include_directories(${CMAKE_BINARY_DIR}/include/)
-# TODO: remove:
-include_directories(${CMAKE_SOURCE_DIR}/src/)
 
-#build_example("triangle" "")
+build_example("fill-rect" "")
```
Comment (dbatyai) [inline examples/CMakeLists.txt]: Please add a short description of what the example does.

File: src/gpu/gles2/gepard-gles2.cpp [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+/*!
+ * \brief GepardEngine::fillRect
+ * \param x  X-axis value of _start_ and _end_ point
+ * \param y  Y-axis value of _start_ and _end_ point
+ * \param w  size on X-axis
+ * \param h  size on Y-axis
+ *
+ * \todo documentation is missing
+ */
+void  GepardEngine::fillRect(float x, float y, float w, float h)
```
Comment (dbatyai) [inline src/gpu/gles2/gepard-gles2.cpp]: extra space after `void`

**Comment by dbatyai** [comment]
LGTM

**Comment by kkristof** [comment]
LGTM

**Pull request merged**